SWEEGO_CONFIGURATION_REQUISE.md

```
@@ -0,0 +1,188 @@
+```env
+# Adresse email autorisée par votre compte Sweego
+SWEEGO_FROM_EMAIL=test@sweego.io
+```
+
+3. Mettre à jour `server/services/emailService.ts` :
+
+```typescript
+const fromEmail = ENV.sweegoFromEmail || 'doriansarry@yahoo.fr';
+```
+
+## 📧 Credentials Sweego Actuelles
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)